react-client/src/App.jsx

```diff
@@ -54,17 +55,32 @@ onChangePassword(e){
   })
 }
 Login(){
-  if(this.state.userName==='raed' && this.state.password==='123'){
+  $.ajax({
```

Comment:
Would suggest using something like axios, which supports the same promise structure that react uses.